openedx/core/djangoapps/persistenttask/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models to support persistent tasks.
+"""
+
+from django.db import models
+from jsonfield import JSONField
+
+
+class FailedTask(models.Model):
+    """
+    Representation of tasks that have failed
+    """
+    task_name = models.CharField(max_length=255)
+    task_id = models.CharField(max_length=255, db_index=True)
+    args = JSONField(blank=True)
+    kwargs = JSONField(blank=True)
+    exc = models.CharField(max_length=255)
+    datetime_failed = models.DateTimeField(db_index=True)
+    datetime_resolved = models.DateTimeField(blank=True, null=True, default=None)
```

Comment:
Should `datetime_resolved` also be indexed?  As you'll be searching by it in the management command to clean up old resolved tasks?